server/routes/conversation.js

```diff
@@ -0,0 +1,17 @@
+const express = require("express");
+const router = express.Router();
+const protect = require("../middleware/auth");
+const {
+  createConversation,
+  getAllConversation,
+  getConversationById,
+} = require("../controllers/conversation");
+
+router.use(protect);
+
+router.route("/").post(createConversation);
+router.route("/conversations/:userId").get(getAllConversation);
```

Comment:
Instead of using req.params.userId, when you use the protect middleware you should be able to access req.user directly so you don't need the additional param

Reply:
Oh, okay I have made the changes !!